.travis.yml

```diff
@@ -12,28 +12,20 @@ env:
   - CONDA_CHANNELS='conda-forge'
 matrix:
   include:
-  - env: PYTHON_VERSION=2.7
+  - env: PYTHON_VERSION=3.8
     os: linux
     language: generic
-  - env: PYTHON_VERSION=2.7
-    os: osx
-    language: generic
-  - env: PYTHON_VERSION=3.6
-    os: linux
-    language: generic
-  - env: PYTHON_VERSION=3.6
+  - env: PYTHON_VERSION=3.8
```

Comment:
Should we also have 3.7? I think it's good to have two versions at least, it's easier to see if a failure is just a glitch.

Reply:
I don't think it is necessary, but I added it just for you :smile: